osu.Game.Rulesets.Catch/Difficulty/CatchDifficultyAttributes.cs

```diff
@@ -27,7 +27,6 @@ public override IEnumerable<(int attributeId, object value)> ToDatabaseAttribute
             // Todo: osu!catch should not output star rating in the 'aim' attribute.
             yield return (ATTRIB_ID_AIM, StarRating);
             yield return (ATTRIB_ID_APPROACH_RATE, ApproachRate);
-            yield return (ATTRIB_ID_MAX_COMBO, MaxCombo);
```

Comment:
This attribute hasn't been removed, just moved from each ruleset down to the base `DifficultyAttributes` class.

The reasoning for why it was this way is that, historically, some rulesets (e.g. mania) didn't store max combo. This has changed over the years and we now store max combo for all rulesets, but this code hadn't been adjusted in line with that.